src/System.Text.Encoding.CodePages/dir.props

```diff
@@ -3,5 +3,6 @@
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
+    <IsUAP>true</IsUAP>
```

Comment:
The .NET Native toolchain has a dependency on this assembly. We detect if the app needs encodings which are not normally supported and add a dependency to this assembly in that case.